lab_5_scrapper/scrapper.py

```diff
@@ -120,45 +252,103 @@ def __init__(self, full_url: str, article_id: int, config: Config) -> None:
         """
         Initializes an instance of the HTMLParser class
         """
-        pass
+        self._full_url = full_url
+        self._article_id = article_id
+        self._config = config
+        self.article = Article(self._full_url, self._article_id)
 
     def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         """
         Finds text of article
         """
-        pass
+        for text in article_soup.find_all('p')[:-2]:
+            self.article.text += text.text + '\n'
 
     def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> None:
         """
         Finds meta information of article
         """
-        pass
+        title = article_soup.find('h1', attrs={'itemprop': 'headline'})
+        if title:
+            self.article.title = title.text
+
+        author = article_soup.find('a', class_='article-info_articleInfoAuthor__W0ZnW')
+        if author:
+            self.article.author = [author.text]
+        if not author:
+            self.article.author = ["NOT FOUND"]
+
+        topics = article_soup.find_all("a", class_='article-tags_articleTagsLink__El86x')
+        if topics:
+            self.article.topics = [topic.text for topic in topics]
+
+        date = article_soup.find('span', attrs={'class': 'article-info_articleInfoDate__S0E0P'})
```

Comment:
consider moving date preprocessing to a separate method

Reply:
done